### PR TITLE
:ambulance: all blogs on websites for admin

### PR DESCRIPTION
**Pull request by KolushovAlexandr**

### CODE REVIEW

File: website_multi_company_blog/doc/changelog.rst [hunk 1]
```diff
@@ -1,3 +1,8 @@
+`1.0.1`
+-------
+
+- **Fix:** all blogs were shown to admin regardless of website
+
```
Comment (kaadevelop) [inline website_multi_company_blog/doc/changelog.rst]: Has to be tested

**Comment by ilmir-k** [comment]
@Ramil-Mukhametzyanov I see your notes here, leave your review to move forward

File: website_multi_company_blog/doc/changelog.rst [hunk 2]
```diff
@@ -1,3 +1,8 @@
+`1.0.1`
+-------
+
+- **Fix:** all blogs were shown to admin regardless of website
```
Comment (Ramil-Mukhametzyanov) [inline website_multi_company_blog/doc/changelog.rst]: :+1: checked

**Comment by KolushovAlexandr** [comment]
@Ramil-Mukhametzyanov I had logged in as Admin and saw this
Screenshot from 2019-09-30 14-51-41

They were unpublished. The 'future of books' after publishing is seen by a public user

**Comment by em230418** [comment]
I approve to merge it now

**Pull request merged**